cluster.go

```diff
@@ -468,6 +470,12 @@ func (c *clusterState) slotClosestNode(slot int) (*clusterNode, error) {
 	return node, nil
 }
 
+func (c *clusterState) slotRandomEligibleNode(slot int) *clusterNode {
```

Comment:
slotRandomEligibleNode -> slotRandomNode, because I don't understand what sense `Eligible` tries to convey here

Reply:
The idea was that it's not a random selection of any cluster node, but just the ones that make sense for that slot. It wasn't a great name, so I've implemented your suggestion in 22d7121.